strings/strings.json

```diff
@@ -7479,5 +7479,9 @@
   "CloseFiltersDropdown": {
     "en": "Close the filters dropdown",
     "fr": "Fermer le menu déroulant de filtres"
+  },
+  "NoValuesFound": {
```

Comment:
I was looking into the strings file to check if we did not have something similar/close enough already translated that we could use, but does not seems like it.

Kind of a bummer :(